python/ray/util/collective/tests/test_collective_single_node_2_gpus.py

```diff
@@ -0,0 +1,267 @@
+"""Test the collective group APIs."""
```

Comment:
hmm, are we gonna run this on Ray CI?

Actually, maybe we can add it on RayCI once we have GPU CI available.

Reply:
yeah, maybe let's look at how to integrate these tests with Ray tests later? I have tested locally on my GPU cluster and all unittests passed here.

Reply:
OK sounds good :)